DHS_Recode_VII.do

```diff
@@ -31,16 +31,18 @@ global OUT "${root}/STATA/DATA/SC/FINAL"
 global INTER "${root}/STATA/DATA/SC/INTER"
 
 * Define path for do-files
-global DO "${root}/STATA/DO/SC/DHS/Recode VII/Git-DHS-Recode-VII"
+global DO "/Users/sunyining/Dropbox/GitHub/DHS-Recode-VII"
 
 * Define the country names (in globals) in by Recode
     
 	do "${DO}/0_GLOBAL.do"
 	
+	global DHScountries_Recode_VII "Senegal2019"

```

Comment:
Please do not apply the survey name to global as discussed in the final pull request.